modules/cco-ccoctl-configuring.adoc

```diff
@@ -37,6 +37,48 @@ endif::alibabacloud[]
 The `ccoctl` is a Linux binary that must run in a Linux environment.
 ====
 
+.Prerequisites
```

Comment:
Should the prerequisites heading be inside the ifdef, so that a blank .Prerequisites heading doesn't show for non-aws-sts?

(I'm actually not sure if it would show or not with nothing under it, but seems like it should be in the ifdef to be safe)

Reply:
Yes! I just thought of this and was about to point it out in Slack :) The whole prereq section is AWS only (for now)